WebAssembly/Runtime/UnmanagedMemory.cs

```diff
@@ -16,8 +17,90 @@ public sealed class UnmanagedMemory : IDisposable
             => typeof(UnmanagedMemory).GetTypeInfo().DeclaredProperties.Where(prop => prop.Name == nameof(Start)).First().GetMethod);
         internal static readonly RegeneratingWeakReference<MethodInfo> GrowMethod = new RegeneratingWeakReference<MethodInfo>(()
             => typeof(UnmanagedMemory).GetTypeInfo().DeclaredMethods.Where(prop => prop.Name == nameof(Grow)).First());
-        private static readonly RegeneratingWeakReference<long[]> emptyPage = new RegeneratingWeakReference<long[]>(() => new long[Memory.PageSize / 8]);
 
+        /// <summary>
+        /// Copies a <paramref name="length"/> of bytes to the <paramref name="destination"/> memory location from the <paramref name="source"/> memory location.
+        /// </summary>
+        /// <param name="destination">The write target memory location.</param>
+        /// <param name="source">The read target memory location.</param>
+        /// <param name="length">The count of bytes to be copied.</param>
+        public delegate void CopyBlockDelegate(IntPtr destination, IntPtr source, uint length);
+        
+        /// <summary>
+        /// Copies a length of bytes to the destination memory location from the source memory location.
+        /// See <see cref="CopyBlockDelegate"/> for details.
+        /// </summary>
+        public static readonly CopyBlockDelegate CopyBlock;
+
+        /// <summary>
+        /// Sets a <paramref name="length"/> of bytes to the given <paramref name="value"/> at a <paramref name="destination"/> memory location.
+        /// </summary>
+        /// <param name="destination">The write target memory location.</param>
+        /// <param name="value">The byte value to use.</param>
+        /// <param name="length">The count of bytes to be copied.</param>
+        public delegate void InitBlockDelegate(IntPtr destination, byte value, uint length);
+
+        /// <summary>
+        /// Sets a length of bytes to the given value at a destination memory location.
+        /// See <see cref="InitBlockDelegate"/> for details.
+        /// </summary>
+        public static readonly InitBlockDelegate InitBlock;
```

Comment:
None of the new code should be part of the public API.
```suggestion
        static readonly InitBlockDelegate InitBlock;
```